packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -945,8 +948,12 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
      * Force release the drag action by code
      */
     public releaseDrag() {
-        this._scaleBoxesDragBehavior.releaseDrag();
-        this._rotateSpheresDragBehavior.releaseDrag();
+        this._scaleBoxesDragBehaviors.forEach((dragBehavior) => {
+            dragBehavior.releaseDrag();
+        });
+        this._rotateSpheresDragBehaviors.forEach((dragBehavior) => {
+            dragBehavior.releaseDrag();
+        });

```

Comment:
Would be good to empty the array afterwards, either here (if it makes sense) or in the dispose function.

You can set the length to 0 to avoid recreating the array

Reply:
it does not. you can release drag multiple times ans arrays are filled in contructor.